web/book/src/tutorial/aggregation.md

```diff
@@ -1,95 +1,98 @@
 ```
 
-... and so on. Of course this is not the way to go, because it is way too
-repetitive and we need to type all of the artist ids by hand. Luckily, `group`
-will do exactly what we want: separate relation into groups, apply a function to
-each of the groups and combine the results back together:
+... and so on. Of course this is repetitive (and boring) and error prone (because we would need to type each `billing_city` by hand). Moreover, we would need to create the `billing_city` list before we started.
+
+**group:** The `group` transform separates the table into groups (say, those having the same city)
+using information that's already in the table.
+It then applies a transform to each group, and combines the results back together:
 
 ```
-from albums
-group artist_id (
-    aggregate {sum_of_ids = sum album_id}
+from invoices
+group billing_city (
+    aggregate {
+        num_orders = count this,
+        sum_of_orders = sum total,
+    }
 )
 ```
 
-People who know how this is done in SQL would probably noticed that we just
-decoupled aggregation from grouping. These two very connected operations benefit
-immensely from each being a standalone function. Firstly, this allows each of
-them to have some invariants that the query engine can leverage to produce more
-efficient query planes. Additionally, they can be used with other transform
-functions:
+People who know how this is done in SQL will probably have noticed that we just
+decoupled aggregation from grouping. These two very connected operations (in SQL) benefit
+immensely from each being a standalone function.
+
+Firstly, each can have invariants that the query engine can
+leverage to produce more efficient queries.
+Additionally, they can be used with other transform functions, such as:
 
 ```
-from albums
-group artist_id (
+from invoices
+group billing_city (
     take 2
 )
 ```
 
-We can derive what this code chunk is doing using the same logic use used
-before: `group` splits the relation into chunks each of which is corresponding
-to some `artist_id`. Then it applies function `take 2`, which will take first
-two rows of the chunk, and at the end it combines the rows back together.
+This code collects the first two rows for each city's `group`.
 
-SQL needed to replicate this behavior includes window functions and multiple
-sub-queries. Some dialects (PostgreSQL, DuckDB, Google BigQuery) have a special
-syntax to improve performance and reduce the query complexity with
-`DISTINCT ON`. This approach only works when we have `take 1` instead of
-`take 2`, so it's not a general case solution.
+The SQL needed to replicate this behavior might include window functions and
+sub-queries. PRQL handles all the complexity.
+Some dialects (PostgreSQL, DuckDB, Google BigQuery) have a special
+syntax to improve performance and reduce the query complexity
+(for example ,`DISTINCT ON` if the query uses `take 1`).
+But it's not a general solution for the case of `take 2`.
```

Comment:
I think this is great but probably belongs in the reference — I liked the guide that @aljazerzen originally posted (which I now can't find...) about the relative uses of each type of guide

Reply:
https://github.com/PRQL/prql/blob/db9ba055721c39dcc73e050736ccfbd9ad718f0e/web/book/src/overview.md

Reply:
Yes, a lot of the side notes may not belong in the Tutorial.  We can evolve their location as we iterate on this.